mw-webapp/src/dataAccessLogic/CurrentProblemDAL.ts

```diff
@@ -55,4 +55,12 @@ export class CurrentProblemDAL {
     await CurrentProblemService.updateCurrentProblemDTO(currentProblemDTO, currentProblem.uuid);
   }
 
+  /**
+   * Delete CurrentProblem by uuid
+   */
+  public static async deleteCurrentProblem(currentProblemUuid: string, dayReport: DayReport) {
+    await CurrentProblemService.deleteCurrentProblemDTO(currentProblemUuid);
+    await DayReportDAL.updateDayReport(dayReport);
```

Comment:
Why can't we call both this methods at the same time, without chaining? Let's use transactions here. The same for similar methods

Reply:
Added batch